src/service/arguments/build/cross-pr-command.ts

```diff
@@ -0,0 +1,19 @@
+import { Command } from "commander";
+import { CommandConstructor } from "@bc/service/arguments/command-constructor";
+import { CLIActionType, BuildActionType } from "@bc/domain/cli";
+
+/**
+ * Create cross pull request flow sub-subcommand for build subcommand
+ * @implements {CommandConstructor}
+ */
+export class CrossPullRequestCommand implements CommandConstructor {
+    createCommand(): Command {
```

Comment:
I think an abstract pull request might be better instead of an abstract build cause from what I saw the branch options seem to be a bit different from the rest of them.